charactersheet/charactersheet/models/common/magic_item.js

```diff
@@ -26,6 +26,13 @@ function MagicItem() {
     self.magicItemTypeOptions = ko.observableArray(Fixtures.magicItem.magicItemTypeOptions);
     self.magicItemRarityOptions = ko.observableArray(Fixtures.magicItem.magicItemRarityOptions);
 
+    self.magicItemTotalWeight = ko.pureComputed(function() {
```

Comment:
Having a variable called `magicItemTotalWeight` and one called `totalMagicItemWeight` is very misleading and confusing. Can we change it to something more meaningful? Or you could just use a ternary operator on line 51. 
`weightTotal += self.magicItems()[i].magicItemWeight() ? self.magicItems()[i].magicItemWeight() : 0;`